modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/checker/tasks/PartitionReconciliationProcessorTask.java

```diff
@@ -212,13 +216,17 @@ public PartitionReconciliationJob(
         @Override public T2<String, ExecutionResult<ReconciliationAffectedEntries>> execute() throws IgniteException {
             Set<String> caches = new HashSet<>();
 
+            Collection<String> cacheNames = F.viewReadOnly(ignite.context().cache().cacheDescriptors().values(),
```

Comment:
Perhaps, you can use `ignite.context().cache().cacheNames();` instead of this.

Reply:
Done